### PR TITLE
Feat/storybook deployments

### PR DESCRIPTION
**Pull request by christiananese**
Moved [#63](https://github.com/Developer-DAO/web3-ui/pull/63) on a branch within the main repo

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -17,3 +17,5 @@ yarn-error.log
 /coverage
 
 dist
+/storybook-static
+build-storybook.log
```
Comment (etr2460) [inline .gitignore:21]: nit: newline

**Review by etr2460 — COMMENTED** [review]
thanks for moving it! i'd recommend making this change too so that chromatic passes: https://github.com/Developer-DAO/web3-ui/pull/79/files#diff-f8399c1f412cdca5940135dde1b12972010414e56513e58429b3a1416e371b10R10

**Review by etr2460 — APPROVED** [review]
looks great! thanks for getting this setup!

**Comment by christiananese** [comment]
> thanks for moving it! i'd recommend making this change too so that chromatic passes: https://github.com/Developer-DAO/web3-ui/pull/79/files#diff-f8399c1f412cdca5940135dde1b12972010414e56513e58429b3a1416e371b10R10

Thank you, had to rebase my branch to see the component

**Comment by etr2460** [comment]
@allcontributors add @christiananese for code

**Pull request merged**